packages/node_modules/@ciscospark/widget-message-meet/src/container.js

```diff
@@ -178,7 +182,14 @@ export class MessageMeetWidget extends Component {
                 image={toUserAvatarUrl}
                 name={toUser.displayName}
                 onMenuClick={this.handleMenuClick}
-              />
+              >
+                {
+                  callStartTime &&
+                  <div className={classNames(`title-bar-call-timer`, styles.callTimer)}>
```

Comment:
We should also figure out a way to make this available in the `widget-meet` so the call timer would be available for anyone just using that widget.